### PR TITLE
Add command for setting supplementary GIDs

### PR DESCRIPTION
**Pull request by GrantSeltzer**
Handles #265

One thing that I questioned adding is having a check for [NGROUPS_MAX ](https://www.j3e.de/ngroups.html), interested to hear thoughts.

Signed-off-by: grantseltzer <grantseltzer@gmail.com>

### CODE REVIEW

File: acbuild/set-supplementary-groups.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+	if debug {
+		stderr("Setting supplementary groups to ", args)
+	}
+
+	var intArgs = []int{}
+	for _, stringArg := range args {
+		intArg, err := strconv.Atoi(stringArg)
+		if err != nil {
+			return getErrorCode(err)
```
Comment (dgonyeo) [inline acbuild/set-supplementary-groups.go]: Can you add a `stderr` call right before this to explain to the user why we're exiting?

**Comment by dgonyeo** [comment]
Thanks so much for the PR! One tiny nit, but aside from that LGTM.

File: acbuild/set-supplementary-groups.go (hunk 1 above)
Comment (GrantSeltzer) [inline acbuild/set-supplementary-groups.go]: Done.

**Comment by GrantSeltzer** [comment]
@dgonyeo No problem, fixed the issue.

**Pull request merged**